BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -52,10 +68,31 @@ typedef UINT8 HASH_CTX[IPP_HASH_CTX_SIZE];   //IPP Hash context buffer
 // RSA verify required key parameter format
 //
 typedef struct {
-  UINT32                   Signature;
-  UINT8                    PubKeyData[0];
+  UINT8                    PubExp[4];
+  UINT8                    Modulus[0];

```

Comment:
Do we really need to define RSA_PUB_KEY ? 
Also better to put Moulus first, then Exp.

Reply:
We could define it as PUB_KEY and other one as PUB_KEY_HDR.

Modulus is not of fixed length and we define [0] size. I think it would be better to PubExp first and then Modulus.